src/isomorphism.rs

```diff
@@ -175,54 +180,70 @@ impl Vf2State {
 ///
 /// * Luigi P. Cordella, Pasquale Foggia, Carlo Sansone, Mario Vento;
 ///   *A (Sub)Graph Isomorphism Algorithm for Matching Large Graphs*
-pub fn is_isomorphic(dag0: &PyDiGraph, dag1: &PyDiGraph) -> PyResult<bool> {
-    let g0 = &dag0.graph;
-    let g1 = &dag1.graph;
+pub fn is_isomorphic<Ty>(
```

Comment:
Is this function used anywhere anymore? Maybe we should just delete it and rely solely on `is_isomorphic_matching` (and maybe rename it to `is_isomorphic`)

Reply:
deleted and renamed.